include/glow/Graph/Nodes.h

```diff
@@ -91,8 +91,20 @@ class Constant : public Storage {
     return k->getKind() == Kinded::Kind::ConstantKind;
   }
 
-  Tensor &getPayload() { return payload_; }
+  /// \returns a mutable reference to the payload tensor. If the payload tensor
+  /// is unowned then it will be converted to an owned copy before returning.
+  Tensor &getPayloadMutable() {
+    // If payload is unowned, make an owned copy of the payload for
+    // modification.
+    if (payload_.isUnowned()) {
+      return payload_ = payload_.clone();
+    }
+    assert(!payload_.isUnowned() &&
```

Comment:
nit: This feels unnecessary, as the above if condition checks the not case and returns immediately.

Reply:
Oh I didn't mean to add the return there, copy paste mistake.
Do you think it's unnecessary to assert at all though? Maybe it is

Reply:
Ah ok. Yeah go ahead and leave it then.